src/styles/normalize.css

```diff
@@ -0,0 +1,10 @@
+.html {
+  font-family: "Roboto", sans-serif;
```

Comment:
you should pick some value for default font size and delete repeats in whole file.